api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -275,10 +259,7 @@ private void sendWithRetries(
             .html(htmlMessage)
             .subject(subject)
             .fromEmail(workbenchConfigProvider.get().mandrill.fromEmail);
-    sendWithRetries(msg, description);
-  }
 
-  private void sendWithRetries(MandrillMessage msg, String description) throws MessagingException {
```

Comment:
beta access was the the only email which called this method, so it can be combined with the above method